src/wallet/wallet.cpp

```diff
@@ -5079,7 +5079,7 @@ void CWallet::postInitProcess(CScheduler& scheduler)
 
     // Run a thread to flush wallet periodically
     if (!CWallet::fFlushScheduled.exchange(true)) {
-        scheduler.scheduleEvery(MaybeCompactWalletDB, 500);
+        scheduler.scheduleEvery(MaybeCompactWalletDB, 500 * 1000);
```

Comment:
Nope, this one is from bitcoin

Reply:
https://github.com/dashpay/dash/pull/2682/commits/1100b4e26b52eba502539e311cee4fd6a1b1e0c2